gzll/CHANGELOG.rst

```diff
@@ -9,10 +9,12 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
-Main branch
-***********
+nRF Connect SDK v2.2.0
+**********************
+
+All the notable changes included in the |NCS| v2.2.0 release are documented in this section.
 
-Support nRF5340 network core.
+* Support nRF5340 network core.
```

Comment:
Suggest removing this 'support nRF534' line instead. The style is thus more consistent with the other nrfxlib changelogs.

Reply:
fixed